pages/1.12/metrics/architecture/index.md

```diff
@@ -7,9 +7,9 @@ excerpt: How DC/OS manages metrics
 enterprise: false
 ---
 
-Metrics in DC/OS, version 1.12 and newer, are based on [Telegraf](https://github.com/dcos/telegraf). Telegraf provides an agent-based service that runs on each master and agent node in a DC/OS cluster. By default, Telegraf gathers metrics from all of the processes running on the same node, processes them, then sends the collected information to a central metrics database. 
```

Comment:
Looks like your code editor chopped the hanging space here - let's not add an unnecessary change to this line.